htop.1.in

```diff
@@ -317,6 +317,17 @@ The library size of the process.
 .B M_DT (DIRTY)
 The size of the dirty pages of the process.
 .TP
+.B M_SWAP (SWAP)
+Size of the process's swapped pages.
+.TP
+.B M_PSS (PSS)
+proportional set size, same as M_RESIDENT but each page is divided by the
+number of processes sharing it.
+.TP
+.B M_M_PSSWP (PSSWP)
+Shows proportional swap share of this mapping, unlike \fBM_SWAP\fR this does not take
```

Comment:
`The proportional swap share...` ?
Also here `M_SWAP` gets formatted bold, but `M_RESIDENT` in `M_PSS` does not; it should probably be consitent

Reply:
Most man pages have the convention to make such references bold, but the htop man page has them not bold (so far). I removed the bold for consistency in this PR. What do you think about making all such references bold (in a separate PR)?